cms/templates/cms/home_page.html

```diff
@@ -117,14 +114,12 @@ <h5 class="upper-case">Cost Savings</h5>
       <div class="quote-section">
         <div class="row quote">
           <img class="person" src="{% static 'images/reif1.png' %}" alt="">
-          <h2 class="quote-text">"We will find people who never thought they would be able to apply to MIT.
-          And they will discover that they are much stronger in the global competition than they think."</h2>
+          <h2 class="quote-text">"We will find people who never thought they would be able to apply to MIT. And they will discover that they are much stronger in the global competition than they think."</h2>
           <p class="quote_author">-- Rafael Reif, MIT President  </p>
         </div>
         <div class="row quote">
           <img class="person" src="{% static 'images/sanjay1.png' %}" alt="">
-          <h2 class="quote-text">"Inverted admissions has the potential to disrupt traditional modes of
-          access to higher education... We’re democratizing access to a master’s program for learners worldwide."</h2>
+          <h2 class="quote-text">"Inverted admissions has the potential to disrupt traditional modes of access to higher education... We’re democratizing access to a master’s program for learners worldwide."</h2>
```

Comment:
Should the "..." (three period characters) actually be "…" (one ellipsis character)? Also curly vs. straight quotes.